Tests/IntegrationTestBundle/Entity/RevalidatePreviousStepsData.php

```diff
@@ -20,26 +17,15 @@ public static function resetValidationCalls() {
 		self::$validationCalls = 0;
 	}
 
-	public function isDataValid(ExecutionContextInterface $context) {
+	/**
+	 * @Assert\Callback(groups={"flow_revalidatePreviousSteps_step1"})
+	 */
+	// TODO should be type-hinted with Symfony\Component\Validator\Context\ExecutionContextInterface, but tests fail with Symfony 2.6.4 and PHP 5.3.3, see https://travis-ci.org/craue/CraueFormFlowBundle/jobs/94383918#L249

```

Comment:
I suggest you just bump the min version to PHP 5.3.9 (as done in Symfony 2.7), to be able to use the new API all the time (on PHP 5.3.3, you can only have either the deprecated API, or the 3.0 API, but not the code implementing the 3.0 API with a BC layer for the 2.4 one)